### ISSUE TITLE
Not all R code chunk options are showed in autocomplete. 

### ISSUE THREAD

**Issue by Fred-Wu** [quarto-dev/quarto#18]

**Comment by jjallaire** [comment]
Yes, in VS Code we are specifically limiting the options to those available in both Jupyter and Knitr (with the rationale that most users are targeting Jupyter and we don't want to show them options that won't be valid).

**Comment by ntluong95** [comment]
Hi, I created an extension called `Quarto Wingman` that lists all chunk options. You can try at: https://marketplace.visualstudio.com/items?itemName=ntluong95.quarto-wingman or https://open-vsx.org/extension/ntluong95/quarto-wingman
